back/app/jobs/image_assignment_job.rb

```diff
@@ -0,0 +1,10 @@
+class ImageAssignmentJob < ApplicationJob
```

Comment:
Why isn't it used anywhere?

Reply:
Sorry, didn't notice the PR in ee.

Reply:
BTW why haven't you added it to ee repo?

Reply:
Indeed I also considered to put it in the citizelab-ee repo. I don't really have a good reason for it, just that I didn't find a good place within the citizelab-ee repo. The only services in the citizenlab-ee repo are from the multi_tenancy engine, and I don't think this is really a multi-tenancy-specify thing.

Reply:
I see, it makes sense. But I still believe as `multi_tenancy` is a module (engine), it should be more independent from the main code.
1. It decreases coupling https://en.wikipedia.org/wiki/Coupling_(computer_programming)
2. It increases information hiding / incapsulation https://en.wikipedia.org/wiki/Information_hiding

A practical example. Somebody can use OS version of citizenlab, modify `ImageAssignmentJob` without adjusting ee. So, ee will be broken.
I would put this file into `back/engines/ee/multi_tenancy/app/jobs/multi_tenancy/`.

But I don't want to insist, the decision is up to you :slightly_smiling_face: